src/sage/misc/latex_standalone.py

```diff
@@ -1539,37 +1541,37 @@ def from_graph(cls, graph, merge_multiedges=True,
         EXAMPLES::
 
             sage: from sage.misc.latex_standalone import TikzPicture
-            sage: g = graphs.PetersenGraph()       # optional sage.graphs
-            sage: tikz = TikzPicture.from_graph(g) # optional sage.graphs dot2tex graphviz
+            sage: g = graphs.PetersenGraph()                                            # needs sage.graphs
+            sage: tikz = TikzPicture.from_graph(g)      # optional - dot2tex graphviz, needs sage.graphs
             doctest:...: FutureWarning: This class/method/function is marked as experimental.
```

Comment:
I would add block-scoped `# needs sage.graphs`.

Reply:
The same comment for other changes below in this file.

Reply:
Done in 9132250a2d